src/mixer.c

```diff
@@ -141,21 +141,22 @@ static void MIXER_UpdateRawInputs()
 {
     int i;
     //1st step: read input data (sticks, switches, etc) and calibrate
-    for (i = 1; i <= NUM_TX_INPUTS; i++) {
-        unsigned mapped_channel = MIXER_MapChannel(i);
-        if (PPMin_Mode() == PPM_IN_TRAIN2 && Model.train_sw && raw[Model.train_sw] > 0) {
+    if (PPMin_Mode() == PPM_IN_TRAIN2 && Model.train_sw && raw[Model.train_sw] > 0) {
```

Comment:
@PhracturedBlue please double check the change in this file. I think your mixer tests cover here. but better you can review this.